bin/dlang/d2-release

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+set -eu
+
+# Paths
+r=$(dirname $0)/../..
+beaver=$r/bin/beaver
+
+# Use github utilities
+. $r/lib/github.sh
+
+# Convert the code to D2
+$beaver make DVER=2 d2conv
+
+set -x
+
+# Commit the changes and tag
+git config user.name="$(git for-each-ref --format="%(taggername)"
```

Comment:
What does this result in exactly?

Reply:
Isn't it obvious? It gets the "tagger name" of the last commit. I can add a comment to clarify. This is so the automatically generated tag is created under the same persona that did the original tag.